registry/config_file_test.go

```diff
@@ -0,0 +1,161 @@
+package registry
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/pkg/homedir"
+)
+
+func pass() {
+	pc := make([]uintptr, 10)
+	runtime.Callers(0, pc)
```

Comment:
:-)    Sort of.  Was tied of trying to come up with descriptive text for testcases when I figured the names of the test should do a good job of it.  So, I decided to see what it would look like if we used reflection to print the "pass" message.  If people like this I can dup it for the integration-cli tests.